### PR TITLE
Continued test coverage

### PR DESCRIPTION
**Pull request by andrewxhill**

### CODE REVIEW

File: App/NativeModules/Textile/__tests__/Api.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+} from '../Api'
+import { 
+    ExternalInvite
+} from '../Model'
+
+const threadId = 'QmdNgTtH468cqZFzXCi4sVSWTbJMWQbhYb8cBVyikP9LzW'
+const threadKey = 'VsHHHz8bC8fu9k78RaX8ujQsUzGzaUxwKJyLFKKDacUZoWJaouGnzUQwgmh5'
+
+jest.mock('NativeModules', () => {
+    return {
```
Comment (andrewxhill) [inline App/NativeModules/Textile/__tests__/Api.ts]: we can move this once i'm done

File: App/NativeModules/Textile/__tests__/Api.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    addThread,
+    addThreadFiles
+} from '../Api'
+import { IDirectory } from '../pb/textile-go'
+
+const threadId = 'QmdNgTtH468cqZFzXCi4sVSWTbJMWQbhYb8cBVyikP9LzW'
+const threadKey = 'VsHHHz8bC8fu9k78RaX8ujQsUzGzaUxwKJyLFKKDacUZoWJaouGnzUQwgmh5'
+const threadName = 'Great Name'
+
+describe('textile api', () => {
```
Comment (andrewxhill) [inline App/NativeModules/Textile/__tests__/Api.ts]: happy to remove these tests completely if it doesn't seem valuable. ended up not tackling sagas today....

Comment (asutula) [reply App/NativeModules/Textile/__tests__/Api.ts]: Lots of good stuff in here, I think it's valuable for sure. We can keep expanding this test to cover all the functions?

Comment (andrewxhill) [reply App/NativeModules/Textile/__tests__/Api.ts]: for sure

**Pull request merged**